cmd/flux/create_image_policy.go

```diff
@@ -94,14 +98,27 @@ func createImagePolicyRun(cmd *cobra.Command, args []string) error {
 		policy.Spec.Policy.SemVer = &imagev1.SemVerPolicy{
 			Range: imagePolicyArgs.semver,
 		}
+	case imagePolicyArgs.alpha != "":
```

Comment:
Should also check (perhaps earlier) `alpha XOR semver` -- i.e., the user supplied _one_ of `--select-alpha`, `--select-semver`.

Reply:
Added this validation